cedana-helm/templates/host-daemonset.yaml

```diff
@@ -40,9 +40,9 @@ spec:
               readOnly: false
           command: ["/bin/sh", "-c"]
           {{ if .Values.daemonHelper.upgradeAndRestart }}
-          args: ["cedana k8s-helper --restart true && sleep infinity"]
+          args: ["cedana k8s-helper --restart true --start-otelcol true && sleep infinity"]
```

Comment:
Should this be optional? I think it makes sense to add something on `values.yaml` to enable/disable this.

Reply:
agreed